test/blackbox-tests/test-cases/pkg/path.t

```diff
@@ -0,0 +1,41 @@
+Things should be the same whether dependencies are specified or not.
+
+  $ mkdir dune.lock
+  $ cat >dune.lock/lock.dune <<EOF
+  > (lang package 0.1)
+  > EOF
+
+If we have a package we depend on
+
+  $ mkdir dependency-source
+  $ cat >dune.lock/dependency.pkg <<EOF
+  > (source (copy $PWD/dependency-source))
+  > EOF
+
+And we have a package we want to build
+
+  $ mkdir test-source
+  $ cat >test-source/parade <<EOF
+  > (=ↀωↀ=)✧  ヾ(*ΦωΦ)ﾉ
+  > EOF
+  $ cat >dune.lock/test.pkg <<EOF
+  > (source (copy $PWD/test-source))
+  > (build
+  >  (system "cat parade"))
+  > EOF
+  $ dune build .pkg/test/target/
+  (=ↀωↀ=)✧  ヾ(*ΦωΦ)ﾉ
+
+Now it fails since adding the dependency modified PATH.
+
+  $ cat >dune.lock/test.pkg <<EOF
+  > (source (copy $PWD/test-source))
+  > ; adding deps breaks cat
+  > (deps dependency)
+  > (build
+  >  (system "cat parade"))
+  > EOF
+  $ dune build .pkg/test/target/
+  File ".pkg/test/_unknown_", line 1, characters 0-0:
+  /usr/bin/sh: line 1: cat: command not found
```

Comment:
This line is different on my machine:
```diff
-  /usr/bin/sh: line 1: cat: command not found
+  /bin/sh: cat: command not found
```

Reply:
Indeed, we can use `command` to make the test a little more resilient. Taken from stack overflow:

```
command -v cat >/dev/null 2>&1 || { echo "no cat" }
```